app/routes/events/components/EventAdministrate/RegistrationTables.tsx

```diff
@@ -230,16 +274,15 @@ export class RegisteredTable extends Component<Props> {
         title: 'Dato',
         dataIndex: 'registrationDate',
         render: (date) => (
-          <Tooltip content={<Time time={date} format="DD.MM.YYYY HH:mm:ss" />}>
-            <Time time={date} format="DD.MM.YYYY" />
+          <Tooltip content={<Time time={date} format="DD/MM/YYYY HH:mm:ss" />}>
+            <Time time={date} format="DD/MM/YYYY" />
```

Comment:
Since this page is in norwegian we should use . for dates https://www.sprakradet.no/sprakhjelp/Skriveregler/Dato/

Reply:
Didn't know this was a thing, but okay I'll change it. Slashes are much better in my opinion though 😬